sbirez/management/commands/importtopics.py

```diff
@@ -27,15 +27,15 @@ def handle(self, mdb_filename, solicitation_name, clear=False, **options):
             raise OSError("MS Access database %s not found" % mdb_filename)
 
         # Confirm that solicitation_name is for solicitation in the database
-        proc = subprocess.Popen(#'ls',
```

Comment:
I'm not sure why this check was removed; granted, it's very hacky (especially with an external call to psql) and could be improved, but since we've got no system for entering solicitation data yet, not knowing what solicitations the data is set up to handle seems like an issue we should address one way or the other.  We could make a ticket for a better way...